### ISSUE TITLE
UnicodeDecode Error

### ISSUE THREAD

**Issue by marionbregeault** [chatcannon/galvani#27]
Hi,
Thanks for your code! I would like to use the first part to convert the MPT file into an array. 
However I get the following error when calling MPTfile , and I really don't understand what to change :( Would you tell me what is going on? 

error line : str3 = lambda b: str(b, encoding='ascii')
error message : UnicodeDecodeError: 'ascii' codec can't decode byte 0xb5 in position 165: ordinal not in range(128) 

Thank you.
Marion

**Comment by chatcannon** [comment]
I wasn't sure what encoding to use because I have never had an MPT file with non-ASCII characters, and I no longer have access to a Bio-Logic instrument to create test files. I thought it would be better to raise an error than to produce nonsense data by choosing the wrong encoding.

Try replacing the `encoding='ascii'` with `encoding='utf8'` and see if that works and the text comes out correctly. If not then try whatever character encoding is most often used in your locale e.g. `'latin1'` for Western Europe.

Alternatively send me the file or attach it to this issue and I will have a go.

**Comment by chatcannon** [comment]
P.S. If `encoding='utf8'` works then tell me so that I can change to that in the next release.

**Comment by linuxrider** [comment]
Had same problem with the attached file.
Had to rename extension due to github limitations.
[galvani test.txt](https://github.com/chatcannon/galvani/files/3141806/galvani.test.txt)

`latin1` is a solution in this case.
Does it make sense to optionally state the encoding as kwarg?
The situation with this file is more complicated though, will open another issue for that.

**Comment by chatcannon** [comment]
> Does it make sense to optionally state the encoding as kwarg?

Yes it does, this will be easier if I no longer have to support both python 2 and python 3 so maybe it is worth bringing forward #15 together with this

**Comment by marionbregeault** [comment]
Hi, Yes 'latin1' worked for me ! But not utf8. Thank you!
